### DISCUSSION TITLE
Upload files though filesystem

### DISCUSSION THREAD

**Discussion by bralbral** [minio/minio#16841] [Q&A] [answered]
Is it possible to upload files to minio directories manually via the file system? Will they be indexed later?

**Comment by ravindk89** [comment, marked as answer]
No - MinIO requires you interface with it through the S3 API.

What you can do instead is use the `mc` utility to copy files to and from MinIO as-needed in a POSIX-like fashion.

**Reply by bralbral** [reply]
It seems it was possible in the old version. In any case, thanks!

**Reply by klauspost** [reply]
Yes, but it posed serious limitations to what could be achieved. 

S3 and Posix are not the same and should not be used interchangeably.
